airbyte-ci/connectors/pipelines/pyproject.toml

```diff
@@ -19,7 +19,7 @@ semver = "^3.0.1"
 airbyte-protocol-models = "*"
 tabulate = "^0.8.9"
 jinja2 = "^3.0.2"
-requests = "^2.28.2"
+requests = "2.28.2"
```

Comment:
For our future ref: the logged issues relating to the conflict with `pydocker` recommend `<2.29.0`